### ISSUE TITLE
Missing activation function

### ISSUE THREAD

**Issue by adamwawrzynski** [Hsankesara/DeepResearch#6]
I think you are missing activation function in the last Dense layer of the model, which in this case should be "softmax", as you are predicting one of given classes of documents. According to the Keras documentation:

>  Activation function to use (see activations). If you don't specify anything, no activation is applied (ie. "linear" activation: a(x) = x).

**Comment by Hsankesara** [comment]
Hey, @adamwawrzynski,

You are right that I didn't add softmax function explicitly. Instead, I have used it in hyperparameters. 
https://github.com/Hsankesara/DeepResearch/blob/master/Hierarchical_Attention_Network/HAN.py#L60
You can check it out using the above link. I did it to make the model more flexible. Thanks for taking your time and go through the code. Please let me know if you have any questions.

**Comment by adamwawrzynski** [comment]
@Hsankesara I see, but You forgot to use this hyperparameter inside creation of your neural network. 'activation' hyperparameter is used only once, during definition. I saw this issue during my research on NLP. I am refering to and using Your implementation of HAN network in my Master's thesis (its released on MIT license).

**Comment by Hsankesara** [comment]
Hey, @adamwawrzynski,

You are right I have missed it. I'll add it as soon as possible. I am glad that you're using my implementation in your thesis and wish you all the best for it.

**Comment by Hsankesara** [comment]
Hey @adamwawrzynski,

I updated the model and added the activation function. Thanks for letting me know about it.

**Comment by adamwawrzynski** [comment]
@Hsankesara now it looks correct. My Master's thesis is uploaded on my github page, if You are interested :)

**Comment by Hsankesara** [comment]
I would love to check it out. Thanks and all the best for the future.
